src/main/kotlin/org/rust/lang/core/psi/ext/RsItemsOwner.kt

```diff
@@ -61,7 +61,7 @@ val RsItemsOwner.expandedItemsCached: RsCachedItems
     get() = CachedValuesManager.getCachedValue(this, EXPANDED_ITEMS_KEY) {
         val namedImports = SmartList<CachedNamedImport>()
         val starImports = SmartList<CachedStarImport>()
-        val macros = SmartList<RsMacro>()
+        val macros = SmartList<RsMacroDefinitionBase>()
```

Comment:
I think we should put macros 2.0 to namedCfgEnabled/namedCfgDisabled because they are just like usual named elements. In this case, they should also work in old name resolution (e.g. if they are defined in a function body). `macros` field could be renamed to `legacyMacros`.